Source/cmake/OptionsGTK.cmake

```diff
@@ -112,6 +111,10 @@ else ()
     WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_RESOURCE_USAGE PRIVATE OFF)
 endif ()
 
+if (USE_ANGLE_WEBGL)
+    WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_WEBGL2 PRIVATE ON)
+endif ()
```

Comment:
This is not ideal because the values of the build options like `USE_ANGLE_WEBGL` are not finalized until after `WEBKIT_OPTION_END`, which is where the depends/conflicts get applied. But after that point would be too late.

There's no great way to do this, even though it should be simple. You _could_ have `ENABLE_WEBGL2` "default" to `ON` and let it get turned off by default by the dependency on `USE_ANGLE_WEBGL`, which is not enabled by default. But that's confusing and awkward.

Alternative: perhaps `USE_ANGLE_WEBGL` will be ready for buildbots soon? If it's ready for bots but not for users, then you could make them both depend on `ENABLE_EXPERIMENTAL_FEATURES`.

Alternative: remove the build flags, since they're not needed to guard any system dependencies? That's probably a lot more than you were signing up for, though.